os/net/netmgr/netmgr_ioctl_lwip.c

```diff
@@ -152,7 +152,7 @@ int _netdev_dhcpc_start(const char *intf)
 
 	NET_LOGKV(TAG, "DHCP client started successfully, waiting IP address (timeout %d msecs)\n", timeleft);
 	while (netifapi_dhcp_address_valid(cnif) != 0) {
-		usleep(100000);
+		usleep(200000);
 		timeleft -= 100;
```

Comment:
Is there any connection between `timeleft` and the `usleep` interval? The usleep is increased to 200ms, but timeleft is decremented by 100 only.

Reply:
Dear @kishore-sn ,
ya this seems a little bit tricky here, this modification's outcome is to double the timeout interval. Or do you suggest I change the CONFIG_LWIP_DHCPC_TIMEOUT in defconfig?